shell/browser/hid/electron_hid_delegate.cc

```diff
@@ -36,6 +36,7 @@ ElectronHidDelegate::~ElectronHidDelegate() = default;
 std::unique_ptr<content::HidChooser> ElectronHidDelegate::RunChooser(
     content::RenderFrameHost* render_frame_host,
     std::vector<blink::mojom::HidDeviceFilterPtr> filters,
+    std::vector<blink::mojom::HidDeviceFilterPtr> exclusion_filters,
```

Comment:
Need new specs for `navigator.hid.requestDevice({ exclusionFilters: [] });`

Reply:
Resolved via https://github.com/electron/electron/pull/33557/commits/6e4b59a4ca033c581f31d1c6456810f15d10c815.